tests/test_config_flag.py

```diff
@@ -56,8 +57,8 @@ def test_dump_config():
     """Test --dump-default-config and --dump-config"""
     output = runner.invoke(proselint, "--dump-default-config")
     assert json.loads(output.stdout) == default
-
+    comm = "--dump-config --config tests/test_config_flag_proselintrc.json"
     output = runner.invoke(
-        proselint, "--dump-config --config tests/test_config_flag_proselintrc.json")
+        proselint, comm)
```

Comment:
Just OOC, why couldn't this have just become this?

```python
    output = runner.invoke(
        proselint,
        "--dump-config --config tests/test_config_flag_proselintrc.json")
```

That's under 80 chars on each line.

(For that matter, it could've been this, too, if it really came down to it. Python supports C-style automatic string-literal concatenation.)
```python
    output = runner.invoke(
        proselint,
        "--dump-config "
        "--config tests/test_config_flag_proselintrc.json")
```